CONTRIBUTING.md

```diff
@@ -1,16 +1,16 @@
 Contributing
 ============
 
-This project is work of [many contributors](https://github.com/dblock/slack-api-ref/graphs/contributors). You're encouraged to submit [pull requests](https://github.com/dblock/slack-api-ref/pulls) and [propose topics](https://github.com/dblock/slack-api-ref/issues).
+This project is work of [many contributors](https://github.com/slack-ruby/slack-api-ref/graphs/contributors). You're encouraged to submit [pull requests](https://github.com/dblock/slack-api-ref/pulls) and [propose topics](https://github.com/dblock/slack-api-ref/issues).
```

Comment:
still has some wrong links (maybe do a global search & replace to catch them all?):
```suggestion
This project is work of [many contributors](https://github.com/slack-ruby/slack-api-ref/graphs/contributors). You're encouraged to submit [pull requests](https://github.com/slack-ruby/slack-api-ref/pulls) and [propose topics](https://github.com/slack-ruby/slack-api-ref/issues).
```

Reply:
I missed these, thanks @rursprung ! @kstole care to fix in a future PR?

Reply:
Oh you're right. I think I just ignored everything in CHANGELOG cuz I thought it was individual contributions.

#60